packages/design-system/src/components/web-components/ds-autocomplete/ds-autocomplete.tsx

```diff
@@ -0,0 +1,109 @@
+import { define } from '../preactement/define';
+import { Autocomplete, AutocompleteItem, AutocompleteProps } from '../../Autocomplete';
+import { parseBooleanAttr } from '../wrapperUtils';
+import { TextField } from '../../TextField';
+import { formAttrs } from '../shared-attributes/form';
+import { UseLabelPropsProps } from '../../Label/useLabelProps';
+import { UseHintProps } from '../../Hint/useHint';
+import { UseInlineErrorProps } from '../../InlineError/useInlineError';
+
+const attributes = [
+  'aria-clear-label',
+  'aria-complete-label',
+  // Using the lowercase HTML attribute name rather than `auto-focus` so it's
+  // more natural. There's no reason for us to worry about name collisions.
+  'autofocus',
+  'class-name',
+  'clear-input-text',
+  'clear-search-button',
+  'value',
+  'items',
+  'loading-message',
+  'loading',
+  'menu-heading-id',
+  'menu-heading',
+  'name',
+  'no-results-message',
+  'root-id',
+  ...formAttrs,
+] as const;
+
+type IncompatibleProps =
+  | 'autoFocus'
+  | 'clearSearchButton'
+  | 'items'
+  | 'label-id'
+  | 'label'
+  | 'loading';
+
+interface WrapperProps
+  extends Omit<UseLabelPropsProps & UseHintProps & UseInlineErrorProps, 'id' | 'inversed'>,
+    Omit<AutocompleteProps, IncompatibleProps> {
+  autofocus?: string;
+  clearSearchButton?: string;
+  value: string;
+  items?: string;
+  loading?: string;
+  menuHeading?: string;
+  menuHeadingId?: string;
+  rootId: string;
+}
+
+const Wrapper = ({
+  autofocus,
+  clearSearchButton,
+  hint,
+  value,
+  items,
+  label,
+  loading,
+  menuHeading,
+  menuHeadingId,
+  rootId,
+  ...otherProps
+}: WrapperProps) => {
+  return (
+    <Autocomplete
+      {...otherProps}
+      label={menuHeading}
+      labelId={menuHeadingId}
+      autoFocus={parseBooleanAttr(autofocus)}
+      clearSearchButton={parseBooleanAttr(clearSearchButton)}
+      id={rootId}
+      items={JSON.parse(items)}
```

Comment:
Since `items` is an optional prop, I'm wondering if there's a chance this could throw an error if items is `undefined`. Perhaps we could add a check or a fallback value to handle that scenario?

Reply:
Oh yeah it crashes hard if no items are provided. But not providing any items kinda defeats the purpose of a combobox. Is the idea that these items can be loaded in async in some cases so we don't want to provide a fallback @kim-cmsds ?

Reply:
Good lines of thought. I'll take a look at this use case and push up changes.